script.js

```diff
@@ -1,4 +1,193 @@
+//global variables
+let playerTurnCounter = 1;
+const playerDice = [];
```

Comment:
since its stores dices of more than 1 player, good to name it in plural form. Eg. `playersDices`